rest-server/auth-server/src/main/scala/com/actionml/authserver/dal/mongo/UsersDaoImpl.scala

```diff
@@ -12,25 +13,48 @@ class UsersDaoImpl(implicit inj: Injector) extends UsersDao with MongoSupport wi
   private val users = collection[UserAccount]("users")
   private implicit val ec = inject[ExecutionContext]
 
-  override def find(id: String): Future[Option[UserAccount]] = {
+  override def find(id: String)(implicit log: LoggingAdapter): Future[Option[UserAccount]] = {
     users.find(equal("id", id))
       .toFuture
-      .recover { case e => e.printStackTrace(); List.empty }
-      .map(_.headOption)
+      .recover { case e =>
+        log.error(s"Can't find user with id $id", e)
+        List.empty
+      }.map(_.headOption)
   }
 
-  override def find(id: String, secretHash: String): Future[Option[UserAccount]] = {
+  override def find(id: String, secretHash: String)(implicit log: LoggingAdapter): Future[Option[UserAccount]] = {
     users.find(and(
       equal("id", id),
       equal("secretHash", secretHash)
     )).toFuture
-      .recover { case _ => List.empty }
-      .map(_.headOption)
+      .recover { case e =>
+        log.error(s"Can't find user with id $id and password hash $secretHash")
+        List.empty
```

Comment:
Are you using a different method for logging than Harness? We should use the same if possible and also naming conventions. In Harness we use the com.typesafe.scalalogging.LazyLogging in some places and akka.event.Logging in other places but never akka.event.LoggingAdapter

Can we consolidate these?

Reply:
migrated to LazyLogging